src/sentry_tracing.h

```diff
@@ -4,6 +4,14 @@
 #include "sentry_slice.h"
 #include "sentry_value.h"
 
+// W3C traceparent header: 00-<traceId>-<spanId>-<flags>
+// length: 00-32char-16char-02char
+#define TRACEPARENT_LEN 55
+
+// sentry-trace header: <traceId>-<spanId>-<sampled>
+// length: 32char-16char-01char
+#define SENTRY_TRACE_LEN 51
+
```

Comment:
These can be moved to the test, as they aren't used elsewhere. This limits exposure and potential conflict (since they are in a header).

Alternatively, you can also use them to size your buffers, since you can still do `char traceparent[TRACEPARENT_LEN + 1]`. However, I would suggest prefixing `TRACEPARENT_LEN` with `SENTRY_` or making it `SENTRY_W3C_TRACEPARENT_LEN` to prevent name conflicts.